hokusai-app-updater/README.md

```diff
@@ -66,3 +47,31 @@ user@artsy:~$ gh auth logout
 ? Are you sure you want to log out of github.com account 'artsyuser'? Yes
 ✓ Logged out of github.com account 'artsyuser'
 ```
+
+## Usage
+
+* Prepare a change script (can be bash, etc.). See [examples](examples/).
```

Comment:
It wasn't clear whether the scripts committed here were reusable or maintained, or whether we should commit all new scripts or just use them once locally. I renamed `update_scripts/` to `examples/` just so it's more obvious that these are examples, not necessarily reusable or maintained, and new additions are welcome if helpful as further examples.